src/test/java/org/ays/utility/AysRandomUtil.java

```diff
@@ -97,4 +98,9 @@ public static int generateSeatingCount() {
         return RANDOM.nextInt(999) + 1;
     }
 
+    public static String generateRandomAlphaSuffix() {
```

Comment:
`Random` yazan metotlardaki `Random` ifadesini kaldırabiliriz, sınıfın adında var olduğu için anlaşılmasında problem olmaz ✅ 
```suggestion
    public static String generateAlphaSuffix() {
```